### DISCUSSION TITLE
Welcome to java-github-action-app Discussions!

### DISCUSSION THREAD

**Discussion by asimmanzoor** [asimmanzoor/java-github-action-app#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by asimmanzoor** [comment]
hey lest continue with adding new feature in upcoming release!

**Reply by sandykhanna** [reply]
Hi @asimmanzoor ,

This is to test and certify that the discussions are working well.

thanks
@sandykhanna
